frontend/public/components/error.tsx

```diff
@@ -1,54 +1,62 @@
 import * as React from 'react';
 import { DocumentTitle } from '@console/shared/src/components/document-title/DocumentTitle';
+import {
+  ErrorState as PfErrorState,
+  ErrorStateProps as PfErrorStateProps,
+} from '@patternfly/react-component-groups';
 import { Trans, useTranslation } from 'react-i18next';
 import {
+  ActionList,
+  ActionListGroup,
+  ActionListItem,
   CodeBlock,
   CodeBlockCode,
-  EmptyState,
-  EmptyStateBody,
   Stack,
   StackItem,
-  EmptyStateActions,
-  Icon,
-  Title,
+  Button,
 } from '@patternfly/react-core';
-import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
-import { t_global_icon_color_status_danger_default as globalDangerColor100 } from '@patternfly/react-tokens';
+import { PathMissingIcon } from '@patternfly/react-icons/dist/dynamic/icons/path-missing-icon';
 
-import PaneBody from '@console/shared/src/components/layout/PaneBody';
-import { PageHeading } from './utils';
 import { useLocation } from 'react-router';
 
-const ErrorComponent: React.FC<ErrorComponentProps> = ({ title, message }) => {
+export const ErrorPage404: React.FC<PfErrorStateProps> = (props) => {
   const { t } = useTranslation();
-  return (
-    <>
-      <PageHeading title={t('public~Error')} hideFavoriteButton />
-      <PaneBody data-test-id="error-page">
-        <PageHeading title={title} centerText />
-        {message && <div className="pf-v6-u-text-align-center">{message}</div>}
-      </PaneBody>
-    </>
+
+  const HomeButton = (
+    <ActionList>
+      <ActionListGroup>
+        <ActionListItem>
+          <Button variant="primary" component="a" href="/">
+            {t('public~Return to home page')}
+          </Button>
+        </ActionListItem>
+        <ActionListItem>
+          <Button variant="secondary" component="a" href="/search">
+            {t('public~Search')}
+          </Button>
+        </ActionListItem>
+      </ActionListGroup>
+    </ActionList>
   );
-};
 
-export const ErrorPage404: React.FC<ErrorPage404Props> = (props) => {
-  const { t } = useTranslation();
   return (
-    <div>
+    <>
       <DocumentTitle>{t('public~Page Not Found (404)')}</DocumentTitle>
-      <ErrorComponent title={t('public~404: Page Not Found')} message={props.message} />
-    </div>
+      <PfErrorState
+        status="none"
+        icon={PathMissingIcon}
+        headingLevel="h1"
+        titleText={t('public~404: Page Not Found')}
+        bodyText={t(
+          "public~We couldn't find that page. Another page may have what you need, so try searching.",
```

Comment:
We don't have traditional search, so I worry this could be confusing to users.  Maybe just cut the second sentence?